crates/nu-command/src/viewers/table.rs

```diff
@@ -695,11 +728,12 @@ fn convert_to_table(
         .into_iter()
         .filter(|header| header != INDEX_COLUMN_NAME)
         .map(|text| {
+            let style = style_computer.compute("header", &Value::nothing(Span::unknown()));
```

Comment:
Maybe to pass an actual `header`/`column` value here so the closure could be used?
*image if someone makes a closure with a string match

Reply:
Like you did bellow already actually.

Reply:
Sure.